python/ray/util/placement_group.py

```diff
@@ -109,12 +116,54 @@ def bundle_specs(self) -> List[Dict]:
         return self.bundle_cache
 
     @property
-    def bundle_count(self):
+    def bundle_count(self) -> int:
         self._fill_bundle_cache_if_needed()
         return len(self.bundle_cache)
 
-    def _get_none_zero_resource(self, bundle: List[Dict]):
-        # Set a mock value to schedule a dummy task.
+    def to_json_serializable(self) -> dict:
+        """Convert this placement group into a dict for purposes of json
+        serialization.
+
+        Used when passing a placement group as an option to a Ray client remote
+        function. See set_task_options in util/client/common.py.
+
+        Return:
+            Dictionary with json-serializable keys representing the placemnent
+            group.
+        """
+        # Placement group id is converted to a hex /string/ to make it
+        # serializable.
+        return {"id": self.id.hex(), "bundle_cache": self.bundle_cache}
+
+    @staticmethod
+    def from_json_serializable(serializable_form: dict):
```

Comment:
```suggestion
    def from_json_serializable(serializable_form: dict) -> PlacementGroup:
```
minor typing thing

Reply:
Thanks!
Looks like in current Python this requires wrapping the annotation in quotes:
https://stackoverflow.com/questions/40049016/using-the-class-as-a-type-hint-for-arguments-in-its-methods

I've done that for this and the "empty" constructor that was already there.